### PR TITLE
DOC-7633 recos for backup validation

### PR DESCRIPTION
**Pull request by ashleykwtam**
Addresses DOC-7633:
- Providing recommendations on how customers should test their backups following an inquiry from a partner from Erlang Systems (supporting Hollywoodbets) 

To consider:
- @dt gives % confidence levels in each option - I've included our take on the confidence levels per Liv's feedback without mention of % confidence

relates to https://github.com/cockroachdb/docs/pull/17629, cut another branch due to the v23.2 docs cut

https://deploy-preview-17773--cockroachdb-docs.netlify.app/docs/dev/backup-validation#recommendations

### CODE REVIEW

**Comment by ashleykwtam** [comment]
@livlobo following up on this for review whenever you get the chance, thanks!

**Review by livlobo — APPROVED** [review]

File: src/current/v23.1/backup-validation.md [hunk 1]
```diff
@@ -15,6 +15,16 @@ You can validate a backup of a [cluster]({% link {{ page.version.version }}/back
 
 The options that give the most validation coverage will increase the runtime of the check. That is, `verify_backup_table_data` will take a longer time to validate a backup compared to `check_files` or `schema_only` alone. Despite that, each of these validation options provide a quicker way to validate a backup over running a "regular" restore.
 
+## Recommendations
+
+Cockroach Labs recommends implementing the following validation plan to test your backups:
+
+1. Very frequent [`schema_only`](#validate-a-backup-is-restorable) restores: Checks your credentials, schema issues, and version compatability.  
+1. Frequent [`verify_backup_table_data`](#validate-backup-table-data-is-restorable) restores: Tests that all data files are present and uncorrupted. 
+1. Somewhat frequent ["regular"]({% link {{ page.version.version }}/restore.md %}) restores: Full restores that completely validates a backup.
```
Comment (MichaelTrestman) [inline src/current/v23.1/backup-validation.md:24]: needs a rewording here for grammar

**Review by MichaelTrestman — APPROVED** [review]
LGTM, with that one nit to fix

**Comment by shannonbradshaw** [comment]
@dt can you provide a review here so we can get this merged?

File: src/current/v23.1/backup-validation.md (hunk 1 above)
Comment (ashleykwtam) [reply src/current/v23.1/backup-validation.md]: tightened the wording around here to show that only a full restore can completely validate a backup

**Review by dt — APPROVED** [review]

**Comment by ashleykwtam** [comment]
TFTR all :)

**Pull request merged**